app/screens/home/account/components/options/user_presence/index.tsx

```diff
@@ -119,23 +137,20 @@ const UserStatus = ({currentUser, style, theme}: Props) => {
     }, []);
 
     return (
-        <MenuItem
-            labelComponent={
-                <StatusLabel
-                    labelStyle={style}
-                    status={currentUser.status}
-                />
-            }
-            leftComponent={
+        <TouchableOpacity
```

Comment:
Should these (now or later) be refactored into using the OptionItem component?

With these I mean the few cases where we used to have MenuItem, and has been changed to something that is not a OptionItem.

Reply:
I think they shouldn't.  The main reason being that the MenuItem component started growing more complex.  

Simple UI => Simple Code => Less Overload to read